wolfcrypt/src/wc_pkcs11.c

```diff
@@ -947,6 +963,7 @@ static int Pkcs11RsaKeyGen(Pkcs11Session* session, wc_CryptoInfo* info)
         { CKA_VERIFY,          &ckTrue,  sizeof(ckTrue)  },
         { CKA_PUBLIC_EXPONENT, &pub_exp, sizeof(pub_exp) }
     };
+    int               pubTmplCnt = sizeof(pubKeyTmpl)/sizeof(*pubKeyTmpl);
```

Comment:
You may get a cast warning here from size_t to int. Recommend adding `(int)` cast here.